pdf-toolbox/src/test/java/com/lowagie/examples/objects/bookmarks/PageLabels.java

```diff
@@ -69,13 +72,11 @@ public static void main(String[] args) {
 
         try (PdfReader reader = new PdfReader("PageLabels.pdf")) {
             final String[] pageLabels = PdfPageLabels.getPageLabels(reader);
-            for (int i = 0; i < pageLabels.length; i++) {
-                System.out.println("pageLabels = " + pageLabels[i]);
-            }
+            assert pageLabels != null;
```

Comment:
This will not work without `assert`s being enabled with JVM argument, so `assertThat(pageLabels).isNotNull()` or smth like this (there definitely should be analog)

Reply:
That's right.
Changed